src/main/resources/chains/private.conf

```diff
@@ -11,6 +11,8 @@
     atlantis-block-number = "0"
     agharta-block-number = "0"
     phoenix-block-number = "0"
+    # FIXME: Once the testnet is up an running we should determine this value
```

Comment:
maybe we should provide a new file especially for the new testnet, WDTY?

Reply:
Should we still keep the configurations for a private testnet? It could get a bit messy given that we have private.conf, private.json and private-testnet.conf

We could have for our chains/ files:
- Keep private-testnet.conf (current private.conf) and private-testnet.json (current private.json), with private-testnet.conf having ecip1098 activated at block 0 as currently
- Add internal-testnet.conf and internal-tesnet.json for our testnet

We could have for our conf/ files:
- Have a testnet-changes.conf file with the testnet configuration overrides (i.e. disabling pruning and discovery)
- Have a private-testnet.conf and a internal-testnet.conf that would just select the correct chain

Reply:
maybe we could discuss this in a call.

Reply:
Conclusion from call: https://github.com/input-output-hk/mantis/pull/681